_redirects

```diff
@@ -33,3 +33,5 @@
 /tutorials/module-system/module-system /tutorials/module-system/deep-dive 301
 
 /permalink/stub-ld /guides/faq#how-to-run-non-nix-executables 301
+/manual/nix/unstable https://hydra.nixos.org/job/nix/master/build.x86_64-linux/latest/download 200
+/manual/nix/development https://hydra.nixos.org/job/nix/master/build.x86_64-linux/latest/download 200
```

Comment:
Testing this locally, the proxying doesn't actually work, it does a full redirect

Reply:
We can reasonably assume this may work live, and if it doesn't it's at least not a degradation. In any case this change is a hard requirement for https://github.com/NixOS/nixos-homepage/pull/1446